tsdb/main.go

```diff
@@ -55,13 +55,21 @@ func main() {
 	flag.Parse()
 
 	// Only try and parse the conf file if it exists
```

Comment:
I thought the next comment made it pretty clear what was going on, which is why I left this comment as it was. I can expand on it though.